portal/server/controllers/utilisation-report-service/utilisation-report-upload/utilisation-report-cell-validators.test.js

```diff
@@ -0,0 +1,581 @@
+const {
+  generateUkefFacilityIdError,
+  generateBaseCurrencyError,
+  generateFacilityUtilisationError,
+  generateTotalFeesAccruedError,
+  generateMonthlyFeesPaidError,
+  generatePaymentCurrencyError,
+  generateExchangeRateError,
+} = require('./utilisation-report-cell-validators');
+
+describe('utilisation-report-cell-validators', () => {
```

Comment:
Great job on the thorough tests

I definitely think it would be nicer if each of these describe blocks had their own test file, to make this file easier to manage and more readable

Honestly I think the `utilisation-report-cell-validators.js` file is too long as well so you could split those into separate files as well, e.g.

```
utilisation-report-cell-validators
|- generate-ukef-facility-id-error.js
|- generate-ukef-facility-id-error.test.js
|- generate-base-currency-error.js
|- generate-base-currency-error.test.js
// etc.
```